server/routerlicious/packages/lambdas/src/alfred/index.ts

```diff
@@ -619,6 +622,34 @@ export function configureWebSocketServices(
 				);
 			}
 
+			// Send signal to room from broadcast-signal endpoint
+			if (httpServer !== undefined && eventEmitter !== undefined) {
+				eventEmitter.on("broadcast-signal", (_, containerUrl) => {
+					const documentId = containerUrl.split("/")[containerUrl.split("/").length - 1];
+					const tenantId = containerUrl.split("/")[containerUrl.split("/").length - 2];
+					const roomFromBroadcastSignal: IRoom = { tenantId, documentId };
```

Comment:
I think this here what we need is validation that the `tenantId` and `documentId` match the ones for the room established for this particular websocket? I.e. lines 382-385. Everything that's being set up in this file seems to me to be related to a specific session/room. The events coming from the new emitter could be targeting any room, but will be seen by potentially many "instances of this file" (one for each different room)...?